packages/dds/map/src/interfaces.ts

```diff
@@ -36,9 +36,9 @@ export interface IValueChanged {
  * @remarks When used as a Map, operates on its keys.
  * @alpha
  */
-// TODO: Use `unknown` instead (breaking change).
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
 export interface IDirectory
+	// TODO: Use `unknown` instead (breaking change).
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
I don't think it's too late, but since this package is effectively deprecated, I believe the general attitude is that it wouldn't be worth making API changes here.

Reply:
But since `Directory` is used by the framework itself, maybe this would merit some TLC.